sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -504,6 +515,7 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
           institutionUserTypes: [
             InstitutionUserTypes.admin,
             InstitutionUserTypes.user,
+            InstitutionUserTypes.readOnlyUser,
```

Comment:
This one will allow the read-only user to access the edit programs form as it's the only way to see some data from the program.